api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/Leave.java

```diff
@@ -18,7 +18,7 @@
  * limitations under the License.
  *******************************************************************************/
 
-package org.eclipse.microprofile.lra.annotation;
+package org.eclipse.microprofile.lra.annotation.ws.rs;
```

Comment:
@mmusgrov According to [this comment](https://github.com/eclipse/microprofile-lra/issues/11#issuecomment-442098892) only `@LRA` and `@NestedLRA` should be covered. I believe we should stay consistent with handling of participant annotations (`@Compensate`, `@Complete`, `@Status`, `@Forget`, `@Leave`).